networkx/algorithms/triads.py

```diff
@@ -263,6 +305,16 @@ def all_triads(G):
     -------
     all_triads : generator of DiGraphs
        Generator of triads (order-3 DiGraphs)
+
+    Examples
+    --------
+    >>> G = nx.DiGraph([(1, 2), (2, 3), (3, 1), (3, 4), (4, 1), (4, 2)])
+    >>> triads = [x for x in nx.all_triads(G)]
+    >>> triads[0].edges()
+    OutEdgeView([(1, 2), (2, 3), (3, 1)])
+    >>> triads[1].edges()
+    OutEdgeView([(1, 2), (4, 1), (4, 2)])
```

Comment:
Instead of creating a list from the generator and showing a few of the instance, we could instead just show all of the triads in the graph by iterating over the generator directly:

```suggestion
    >>> for triad in nx.all_triads(G):
    ...     print(triad.edges)
    [(1, 2), (2, 3), (3, 1)]
    [(1, 2), (4, 1), (4, 2)]
    [(3, 1), (3, 4), (4, 1)]
    [(2, 3), (3, 4), (4, 2)]
```

Just an idea though :) If you prefer to stick with the list approach, then I suggest replacing the list comprehension with the simpler `triads = list(nx.all_triads(G))`

Reply:
Yes , thanks I will keep this in mind :)